codebasin.py

```diff
@@ -66,23 +67,25 @@ def guess_project_name(config_path):
         dest="config_file",
         metavar="FILE",
         action="store",
-        help="configuration file (default: <DIR>/config.yaml)",
+        help="Configuration file (default: <DIR>/config.yaml)",
     )
     parser.add_argument(
         "-v",
         "--verbose",
         dest="verbose",
         action="count",
         default=0,
-        help="increase verbosity level",
+        help="""
+    Increase verbosity level""",
     )
     parser.add_argument(
         "-q",
         "--quiet",
         dest="quiet",
         action="count",
         default=0,
-        help="decrease verbosity level",
+        help="""
+    Decrease verbosity level""",
```

Comment:
This inserts a newline, but doesn't seem to do so consistently.

The "Increase" and "Decrease" lines have _extra_ indentation compared to other options:
```
  -r DIR, --rootdir DIR
                        Set working root directory (default .)
  -c FILE, --config FILE
                        Configuration file (default: <DIR>/config.yaml)
  -v, --verbose
                            Increase verbosity level
  -q, --quiet
                            Decrease verbosity level
```